app/src/lib/stores/app-store.ts

```diff
@@ -4652,13 +4663,28 @@ export class AppStore extends TypedBaseStore<IAppState> {
     return Promise.resolve()
   }
 
-  public async _setHideWhitespaceInDiff(
+  public async _setHideWhitespaceInChangesDiff(
+    hideWhitespaceInDiff: boolean,
+    repository: Repository
+  ): Promise<void> {
+    setBoolean(hideWhitespaceInChangesDiffKey, hideWhitespaceInDiff)
+    this.hideWhitespaceInChangesDiff = hideWhitespaceInDiff
+
+    if (enableHideWhitespaceInDiffOption()) {
```

Comment:
Curious if we need the feature flag check here. Wouldn't this only have impact if the user has access to the setting in the UI (which is feature flagged)?

Reply:
yea you shouldn't be able to call this function without the UI, I'll remove this check